src/app/features/container/get-popup-header.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Temporary home for poupup header logic + documentation
```

Comment:
Trying to understand notes/changes here, is this something we are good with at this point or need help?

Reply:
There's a lot of garbage in here, sorry! In the design we have designs for popup headers but they aren't applied to real world examples so I winged it by using `Test-App` to help and made some best guesses.

These notes are my rough work on what new header I decided to use for each popup. 

I have a big comment [here](https://github.com/leather-wallet/extension/issues/4371#issuecomment-1919114939) to replace it so I can delete this. 

I amn't sure if that comment was understood / tested. Thinks look OK from my testing and I believe this work actually relates to the new UX approval project so we could be good as things stand. 

The note for this for example:
`TestAPP -> Profile -> Update Profile` thats the buttons I pressed in `test-app` to generate the popup in question. 

I categorised them the best I could, comparing what we have in prod and what I think I should show from the design.